### PR TITLE
NoTask - Added demographics classify endpoint

### PR DESCRIPTION
**Pull request by benSlaughter**
Added the /admin/demographics/classify endpoint

```
Ran rubocop and rspec
Tested locally
```

### CODE REVIEW

File: lib/bwapi/client/admin/demographics.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module BWAPI
+  class Client
+    module Admin
+      # Search module for admin/demographics endpoints
```
Comment (jonathanchrisp) [inline lib/bwapi/client/admin/demographics.rb]: Not sure we should state search module here, I think we should call it `Demographics module`.

**Comment by jonathanchrisp** [comment]
Looks great, one minor comment but good to go once fixed :)

File: lib/bwapi/client/admin/demographics.rb (hunk 1 above)
Comment (benSlaughter) [inline lib/bwapi/client/admin/demographics.rb]: Copy and paste error :(

**Comment by benSlaughter** [comment]
Fixed typo, going to merge as it is needed urgently

**Pull request merged**